vocabularies/Org.OData.Aggregation.V1.md

```diff
@@ -16,69 +16,109 @@ Term|Type|Description
+[AvailableOnAggregates](./Org.OData.Aggregation.V1.xml#L309:~:text=<Term%20Name="-,AvailableOnAggregates,-")|[AvailableOnAggregatesType](#AvailableOnAggregatesType)|<a name="AvailableOnAggregates"></a>This action or function is available on aggregated entities if the `RequiredProperties` are still defined
 
 
 ## Functions
 
-### <a name="isroot"></a>[isroot](./Org.OData.Aggregation.V1.xml#L216:~:text=<Function%20Name="-,isroot,-")
+### <a name="move"></a>[move](./Org.OData.Aggregation.V1.xml#L226:~:text=<Function%20Name="-,move,-")
 
-Returns true, if and only if the value of the node property of the specified hierarchy is the root of the hierarchy
+Returns an entity in the specified entity set with the same structure as the binding parameter, or null if no such entity exists
+
+The binding parameter "moves into" the specified entity set. This can be used to specify
+a recursive hierarchy from a "directory" of hierarchies, for use in the hierarchy
+functions defined in this vocabulary, for example:
+```
+GET ~/Sales?$filter=SalesOrganization
+ /Aggregation.move(EntitySet=@PathExpression)
+ /Aggregation.isdescendant(Hierarchy='SalesOrgHierarchy',Node='EMEA')
+&@PathExpression=$root/SalesOrgHierarchyDirectory('Regional')/Nodes
+```
 
 Parameter|Type|Description
 :--------|:---|:----------
-**[Entity](./Org.OData.Aggregation.V1.xml#L218:~:text=<Function%20Name="-,isroot,-")**|EntityType|**Binding parameter**
-[Hierarchy](./Org.OData.Aggregation.V1.xml#L219:~:text=<Function%20Name="-,isroot,-")|String|
-[&rarr;](./Org.OData.Aggregation.V1.xml#L220:~:text=<Function%20Name="-,isroot,-")|Boolean|
+**[Entity](./Org.OData.Aggregation.V1.xml#L239:~:text=<Function%20Name="-,move,-")**|EntityType?|**Binding parameter**
+[EntitySet](./Org.OData.Aggregation.V1.xml#L240:~:text=<Function%20Name="-,move,-")|\[EntityType\]|A named or implicit entity set, specified with a path expression
+[&rarr;](./Org.OData.Aggregation.V1.xml#L243:~:text=<Function%20Name="-,move,-")|EntityType?|
+
 
+### <a name="move"></a>[move](./Org.OData.Aggregation.V1.xml#L245)
 
-### <a name="isdescendant"></a>[isdescendant](./Org.OData.Aggregation.V1.xml#L223:~:text=<Function%20Name="-,isdescendant,-")
+Like `move(Edm.EntityType)`, but applied to each entity in a collection
 
-Returns true, if and only if the value of the node property of the specified hierarchy is a descendant of the given parent node with a distance of less than or equal to the optionally specified maximum distance
+This can be used as a set transformation before a hierarchical transformation in `$apply`, for example:
+```
+GET ~/SalesOrganizations?$apply=
+ Aggregation.move(EntitySet=@PathExpression)
+ /descendants(SalesOrgHierarchy,filter(Name eq 'EMEA'))
+&@PathExpression=$root/SalesOrgHierarchyDirectory('Regional')/Nodes
+```
```

Comment:
Using `move` to specify the hierarchy directory in this way changes the
`@odata.context` of the result set to `$metadata#SalesOrgHierarchyDirectory('Regional')/Nodes`. This could be compensated for by adding
```
...
 /Aggregation.move(EntitySet=@OriginalEntitySet)
...
&@OriginalEntitySet=$root/SalesOrganizations
```